### PR TITLE
Fix YouTube subscriber count

### PR DESCRIPTION
**Pull request by TobiGr**
Modify test to fail on too small subscriber count

Fixes TeamNewPipe/NewPipe#2632

CC @theScrabi @Stypox

### CODE REVIEW

File: extractor/src/main/java/org/schabi/newpipe/extractor/utils/Utils.java [hunk 1]
```diff
@@ -27,6 +27,35 @@ public static String removeNonDigitCharacters(String toRemove) {
...
+     *     <li>123 -&gt; 123</li>
+     *     <li>1.23K -&gt; 1230</li>
+     *     <li>1.23M -&gt; 1230000</li>
+     * </ul>
+     * @param numberWord string to be converted to a long
+     * @return a long
+     * @throws NumberFormatException
+     * @throws ParsingException
+     */
+    public static long mixedNumberWordToLong(String numberWord) throws NumberFormatException, ParsingException {
```
Comment (Stypox) [inline extractor/src/main/java/org/schabi/newpipe/extractor/utils/Utils.java]: Since this is a public method you could add a test only for this one

Comment (TobiGr) [reply extractor/src/main/java/org/schabi/newpipe/extractor/utils/Utils.java]: Done. just noticed that there are no tests for the util methods.

**Comment by mauriciocolli** [comment]
Just a question: Shouldn't we have a way to indicate when we _don't_ have a precise number available? It's a shame if someone is using this library as their source of information for some project that depends on it (e.g. track subscribers over time).

Keep in mind that this change would just make the wording/results more accurate, the user would still have to live with that.

Maybe including a warning in the documentation is enough?

**Comment by TobiGr** [comment]
> Shouldn't we have a way to indicate when we _don't_ have a precise number available?

Yes, you are right. A warning in the docs is the minimum, maybe printing a warning to the console would be a good compromise, but even a new method (something like `isSubscriberCountAccurate()`) is possible.
@Stypox @theScrabi

**Pull request merged**